manifests/kustomize/base/pipeline/multi-user/view-edit-roles.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
@elikatsis Can you add an inline comment with short explanation? I think upcomers will be confused too, so it'll be great as a comment.